integrations/tensorflow/WORKSPACE

```diff
@@ -7,7 +7,7 @@
 
 load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 
-TENSORFLOW_COMMIT = "eece4dba1fc65f7977085581852b0d6e6d42f04e"
+TENSORFLOW_COMMIT = "67ba341c869e30ee4a89e040cd875d12b9bc666e"
```

Comment:
(just commenting here for threading)

The change to mlir-hlo includes the stablehlo test suite (https://github.com/tensorflow/mlir-hlo/tree/master/stablehlo/stablehlo/testdata), which we might want to slice out if it increases git checkout time... this is a lot of files: 
<img width="631" alt="image" src="https://user-images.githubusercontent.com/4010439/224433332-ce944103-36cf-4921-b278-f5599eb82170.png">